scrapy/settings/__init__.py

```diff
@@ -368,11 +369,31 @@ def __iter__(self):
     def __len__(self):
         return len(self.attributes)
 
-    def __str__(self):
-        return str(self.attributes)
```

Comment:
current shell implementation uses [`b.append("  %-10s %s" % (k, v))`](https://github.com/scrapy/scrapy/blob/master/scrapy/shell.py#L141), so `__str__` as a dict would be the same as in #1732